src/WebJobs.Script.WebHost/WebHostServiceCollectionExtensions.cs

```diff
@@ -111,6 +111,7 @@ public static void AddWebJobsScriptHost(this IServiceCollection services, IConfi
                 else if (SystemEnvironment.Instance.IsLinuxAppService())
                 {
                     var hostNameProvider = p.GetService<HostNameProvider>();
+                    var functionsHostingConfiguration = p.GetService<IOptions<FunctionsHostingConfiguration>>().Value;
```

Comment:
Not seeing this being used?

Reply:
Fixed this. Thank you for calling it out. Didn't get saved before I committed.